components/channel_header_dropdown/channel_header_dropdown_items.js

```diff
@@ -203,6 +204,13 @@ export default class ChannelHeaderDropdown extends React.PureComponent {
                         dialogProps={{channel}}
                         text={localizeMessage('channel_header.setHeader', 'Edit Channel Header')}
                     />
+                    <Menu.Group divider={divider}>
+                        <MenuItemDirectLevelChannel
+                            id='channelDirectLevel'
```

Comment:
This `id` could be renamed with a more useful name, however, it's only used for testing.